doc/nrf/ug_zigbee_configuring_components.rst

```diff
@@ -0,0 +1,31 @@
+﻿.. _ug_zigbee_configuring_components:
+
+Configuring Zigbee components in |NCS|
+######################################
+
+The Zigbee protocol in |NCS| can be customized by enabling and configuring several components.
+
+This page lists options and steps required for configuring the available components:
+
+.. contents::
+    :local:
+    :depth: 1
+
+.. _ug_zigbee_configuring_components_ota:
+
+Configuring application helper functions
```

Comment:
@wbober , any feedback about this section? It needs new NCS input.

Reply:
@tomchy or @sebastiandraus please provide your input on KConfig options for default signal handler.

Reply:
Talked to Tomasz.